api-docs/generate-api-docs.sh

```diff
@@ -41,7 +41,7 @@ weight: 304
 "
 
   # Use Redoc to generate the v2 API html
-  redoc-cli bundle -t template.hbs \
+  npx redoc-cli bundle -t template.hbs \
```

Comment:
@sanderson Should we give `/api-docs` it's own package.json as well? I also need to update the README with npx.

Reply:
That's not a bad idea. That way those who don't need to generate the API docs won't need to download all the deps to do it. We'll need to update the deploy steps `circleci/config.yml` as well.

Reply:
Done